ONSdigital/response-operations-ui
# Disable webassets caching in testing and debug environment

# Motivation and Context
Currently `response-operations-ui` is set up to cache web assets in all environments.

This raises two issues:
* There's no need to cache in development/debug
* Caching can cause file handling errors in running acceptance tests _concurrently_, which is a problem

# What has changed
If the `Config` object has either the `DEBUG` or `TESTING` flag set `True`, the caching will be disabled on start.

This helps stabilise the acceptance tests, and should also have a slight advantageous affect on development of the frontend.

# How to test?
Run response-ops and confirm that it functions, unit and acceptance tests still pass.

# Links
* [WebAssets Manifest setting](https://webassets.readthedocs.io/en/latest/environment.html#webassets.env.Environment.manifest)
* [WebAssets Cache setting](https://webassets.readthedocs.io/en/latest/environment.html#webassets.env.Environment.cache)
